# A service call for selecting a grasp based on the output of the agile_grasp package and the current end effector pose.
# The request send to the service

geometry_msgs/Pose hand_pose

---

# The response returned by the service

grasp_selection/GraspList grasps
